ee/tabby-webserver/src/service/background_job/mod.rs

```diff
@@ -66,8 +66,8 @@ pub async fn start(
                             }
                         },
                         BackgroundJobEvent::SchedulerGitRepository(repository_config) => {
+                            let mut job_logger = JobLogger::new(SchedulerGitJob::NAME, Some(&repository_config.git_url), db.clone()).await;
```

Comment:
```suggestion
                            let mut job_logger = JobLogger::new(SchedulerGitJob::NAME, serde_json::to_string(event), db.clone()).await;
```

I recommend just use serialized `BackgroundJobEvent` to simplify (so we don't have to remember which job has what param everywhere)

Reply:
For better type safety I think I'll have them pass a `BackgroundJobEvent` and introduce a method which gets the stringified param